src/vm/clrvarargs.cpp

```diff
@@ -33,10 +33,14 @@ void VARARGS::MarshalToManagedVaList(va_list va, VARARGS *dataout)
 {
     WRAPPER_NO_CONTRACT
 
+#if !(defined(_TARGET_ARM_) && defined(PLATFORM_UNIX))
```

Comment:
I don't see why it would be different on ARM from the other platforms on Unix. It rather looks like it is broken for Unix on all platforms due to the different representation of the va_list (on Windows, va_list is just a pointer to the first argument and the arguments are stored consecutively in memory while on Unix, it is an opaque structure).
@jkotas, is CoreCLR actually using this stuff?